include/mitsuba/core/object.h

```diff
@@ -115,6 +115,25 @@ class MTS_EXPORT_CORE Object {
      */
     virtual std::string to_string() const;
 
+
+    /**
+     * \brief Suspend or resume tracking of derivatives.
+     *
+     * One important use case of Mitsuba 2 entails the propagation of derivatives
```

Comment:
Weird paragraph shape, each line is longer than the previous one :-). Can you wrap to 80cols?